specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+Legacy-Regular.ttf      | Legacy       | 400       | 5         | 0      | Legacy        | 400   | 100   | 0     | 0
+Legacy-Bold.ttf         | Legacy       | 700       | 5         | 0      | Legacy        | 700   | 100   | 0     | 0
+Legacy-Black.ttf        | Legacy       | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+Legacy-Soft.ttf         | Legacy Soft  | 400       | 5         | 0      | Legacy        | 400   | 100   | 0     | 0
+Legacy-SoftBold.ttf     | Legacy Soft  | 700       | 5         | 0      | Legacy        | 700   | 100   | 0     | 0
+Legacy-SoftBlack.ttf    | Legacy Soft  | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+
+The "Legacy" typographic family has three weights and each weight has regular and "Soft" variants.
+However, these fonts predate OpenType 1.8, so there is no "SOFT" axis, or any other axis that cannot
+be derived from weight, stretch, and style. For each weight, there are two fonts with the same axis
+values. Therefore, it is not possible to unambiguously select the above fonts using the typographic
+family name and axis values alone.
+
+The hybrid font selection model mitigates this by allowing the specified family name to be _either_
+a typographic family name or a weight-stretch-style name. (It can also be an RBIZ name or full name.)
+In the example above, an application can avoid ambiguity by specifying "Legacy" or "Legacy Soft"
+as the family name. These two cases are handled as follows:
+
+  - "Legacy Soft" does not match a typographic family name, so only fonts in the WWS family
+    are considered. These can be differentiated by weight, so there is no ambiguity.
+
+  - "Legacy" matches both a typographic family and a WWS family. All fonts in the typographic
+    family are considered, but presence in the WWS family is used as a tie-breaker. For example,
+    Legacy-Regular.ttf has the same weight as Legacy-Soft.ttf, but the former is preferred because
+    it is in the "Legacy" WWS family.
+
+Document and application compatibility issues are mitigated by allowing different types of font
+names to be specified and also by providing a method of deriving axis values from weight, stretch,
+and style parameters.
+
+Suppose a document specifies a family name, weight, stretch, and style, but no axis values. The
+application can first map the weight, stretch, style, and font size to axis values by calling the
+`IDWriteFontSet4::GetDerivedFontAxisValues` method. The application can then pass both the family
+name and axis values to `IDWriteFontSet4::GetMatchingFonts`. `GetMatchingFonts` returns a list of
+matching fonts in priority order, and the result is appropriate whether the specified family name
+is a typographic family name, weight-stretch-style family name, RBIZ family name, or full name.
+If the specified family has an "opsz" axis, the appropriate optical size is automatically selected
```

Comment:
"If the specified family has an 'opsz' axis..." and no specific opsz value is specified?